Sources/MongoDriver.swift

```
@@ -20,7 +21,9 @@ public class MongoDriver: Fluent.Driver {
         the given database name, credentials, and port.
     */
     public init(database: String, user: String, password: String, host: String, port: Int) throws {
-        let server = try Server("mongodb://\(user):\(password)@\(host):\(port)", automatically: true)
+        let escapedUser = user.addingPercentEncoding(withAllowedCharacters: .urlHostAllowed)!
+        let escapedPassword = password.addingPercentEncoding(withAllowedCharacters: .urlHostAllowed)!
```

Can we throw an error instead of implicitly unwrapping the password?